README.md

```diff
@@ -1,58 +1,86 @@
 # Storage Extension Specification
 
 - **Title:** Storage
-- **Identifier:** <https://stac-extensions.github.io/storage/v1.0.0/schema.json>
+- **Identifier:** <https://stac-extensions.github.io/storage/v2.0.0/schema.json>
 - **Field Name Prefix:** storage
-- **Scope:** Item, Collection
+- **Scope:** Item, Catalog, Collection
 - **Extension [Maturity Classification](https://github.com/radiantearth/stac-spec/tree/master/extensions/README.md#extension-maturity):** Pilot
 - **Owner**: @davidraleigh @matthewhanson
 
 This document explains the Storage Extension to the [SpatioTemporal Asset Catalog](https://github.com/radiantearth/stac-spec) (STAC) specification.
-It allows adding details related to cloud storage access and costs to be associated with STAC Assets.
+It allows adding details related to cloud object storage access and costs to be associated with STAC Assets.
 This extension does not cover NFS solutions provided by PaaS cloud companies.
 
 - Examples:
```

Comment:
I think an example combining `storage` and `authentication` might be relevant, especially given they use similar `auth:schemes`/`auth:refs` constructs and might often be involved together, such as in `s3` case.

Side note: somewhat annoying that `auth:scheme: ["s3"]` vs `storage:scheme: ["AWS"]` don't align, both in terms of casing, but also for specific vs generic storage reference.

Reply:
I've updated the collection example to use the authentication extension.